content/news/2023-02-24-paper-bray-planemo/index.md

```diff
@@ -0,0 +1,22 @@
+---
+title: 'New Paper "The Planemo toolkit for developing, deploying, and executing scientific data analyses in Galaxy and beyond"'
```

Comment:
@sebastian-schaaf this paper deserves much more attention. Maybe a blog post more about the content with the main figure and only mention this is also published.